ui/components/ui/text-field/text-field.component.js

```diff
@@ -203,6 +205,7 @@ const getBorderedThemeInputProps = ({
       max,
       autoComplete,
     },
+    disableUnderline: 'true',
   },
```

Comment:
This component is deprecated, i created a [ticket](https://consensyssoftware.atlassian.net/browse/MMASSETS-393) to refactor the search bar in setting to use the `TextFieldSearch` component instead;
With @amandaye0h  we thought it makes sense to make these changes globally to all components